tst/testmanuals.g

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+##
+#W  testmanuals.g
+##
+##  <#GAPDoc Label="[1]{teststandard.g}">
+##  <#/GAPDoc>
+##
+
+# This code extracts the examples from manuals chapter-wise and
+# stores this in a workspace.
+
+Print("Extracting manual examples...\n");
+Read(Filename(DirectoriesLibrary("doc/ref"), "makedocreldata.g"));
+
+GAPInfo.ManualDataRef.pathtodoc := DirectoriesLibrary("doc/ref");
```

Comment:
Minor nitpick: `DirectoriesLibrary("doc/ref");` occurs twice, perhaps reorder and use `GAPInfo.ManualDataRef.pathtodoc` in the `Read` ? Not that it matters much...